jwt/jwt.hex

```diff
@@ -1 +1 @@
-184F01E69FF6EE5DDA4BE1D37D531533B39ACD5293CED4799B05044561A165AF
\ No newline at end of file
+7074a5bf6bd6dae368fa598249d57edfcbccc67a1205b2c8d5d2fe7b800663aa
```

Comment:
this enables the cluster BN (lighthouse) to use node repo's geth. makes it easier to debug and reuse running containers

Reply:
https://github.com/ObolNetwork/charon-distributed-validator-node/blob/main/jwt/jwt.hex#L1